### PR TITLE
MKLDNN BoundedRelu implementation for Relu6

### PR DESCRIPTION
**Pull request by pruthviIntel**
1. Added MKLDNNN BoundedRelu op support for Relu6
2. CpuLayout && CPU assignment pass for BoundedRelu Op
3. Unit test inter v/s CPU for BoundedReluOp
4. MKLDNN and default emitter code for BoundedReluOp

### CODE REVIEW

**Comment by pruthviIntel** [comment]
```
(.venv_mxnet) phthoreho@phthoreho-X9DAi:/media/phthoreho/61279ac5-fee9-489e-8593-4891a74aa922/ngraph/build1$ ./test/unit-test --gtest_filter="cpu_fusion.fuse_bounded_relu*"
Note: Google Test filter = cpu_fusion.fuse_bounded_relu*
[==========] Running 2 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 2 tests from cpu_fusion
[ RUN      ] cpu_fusion.fuse_bounded_relu
Comparing 24 bytes[       OK ] cpu_fusion.fuse_bounded_relu (42 ms)
[ RUN      ] cpu_fusion.fuse_bounded_relu_inter_vs_cpu
Comparing 24 bytesmkldnn_verbose,exec,eltwise,jit:sse42,forward_training,fdata:nc fdiff:undef,alg:eltwise_bounded_relu,mb2ic3ih22094iw2097608448,7.80493
[       OK ] cpu_fusion.fuse_bounded_relu_inter_vs_cpu (3969 ms)
[----------] 2 tests from cpu_fusion (4011 ms total)

[----------] Global test environment tear-down
[==========] 2 tests from 1 test case ran. (4011 ms total)
[  PASSED  ] 2 tests.

```

**Comment by pruthviIntel** [comment]
@jbobba addressed the changes requested.

File: src/ngraph/graph_util.hpp [hunk 1]
```diff
@@ -132,5 +132,7 @@ namespace ngraph
 
     bool is_one(std::shared_ptr<Node> reduce_constant);
 
+    bool is_six(std::shared_ptr<Node> reduce_constant);
```
Comment (jbobba) [inline src/ngraph/graph_util.hpp]: remove.

File: src/ngraph/runtime/cpu/pass/cpu_fusion.cpp [hunk 2]
```diff
@@ -1299,3 +1302,51 @@ void ngraph::runtime::cpu::pass::CPUFusion::construct_sigmoid_multiply()
...
+        {
+            return false;
+        }
+        if (pattern_map[alpha]->get_shape() != pattern_map[relu_input]->get_shape())
+        {
+            return false;
+        }
+
+        auto alpha_const_op = std::dynamic_pointer_cast<op::Constant>(pattern_map[alpha]);
+        float alpha_val = *(static_cast<float const*>(alpha_const_op->get_data_ptr()));
```
Comment (Krovatkin) [inline src/ngraph/runtime/cpu/pass/cpu_fusion.cpp]: we are only checking the very first element of a constant. In some obscure corner case, other elements might not necessarily be equal to the first element. Do we care about such a case? @pruthviIntel @jbobba ?

**Pull request merged**